packages/server-admin-ui/src/styles/_bootstrap-variables.scss

```
@@ -61,11 +61,196 @@ $theme-colors: (
 $enable-transitions: true;
 $enable-rounded: true;
 
-// Body
 //
-// Settings for the `<body>` element.
+// Signal K Light Mode Colors
+//
+
+$sk-body-tertiary-bg-light: #e4e5e6;
+
+$sk-primary-light: $blue;
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)